2020/join/gatsby-new-join/src/components/layout.css

```diff
@@ -0,0 +1,344 @@
+}
+
+section.utb {
+  background-color: #446CB3;
+  background: linear-gradient(rgba(68, 108, 179, 0.8), rgba(68, 108, 179, 0.8)), url('https://snworksceo.imgix.net/dpn/4a08586d-e444-4f4b-82e6-0ded406a2e27.sized-1000x1000.jpg');
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  color: #FFF;
+}
+
+section.utb a {
+  color: #CCC;
+}
+
+section.utb a:hover {
+  color: #999;
+}
+
+section.join {
+  background-color: #EEE;
+  height: calc(100vh - 100px);
+}
+
+p.overall {
+  margin: 20px auto;
+  text-align: center;
+}
+
+.join-button {
+  position: fixed;
+  bottom: 20px;
+  right: 20px;
+  padding: 10px;
+  border: 1px solid #AA1E22;
+  border-radius: 2px;
+  background-color: #AA1E22;
+  color: #FFF;
+  cursor:pointer;
+}
+
+.join-button:hover {
+  color: #FFF;
+  border: 1px solid #FFF;
+}
+
+.logo {
+  margin-top: 5px;
+}
+
+.intro-text {
+  color: #FFF;
+  text-align: center;
+  align-self: flex-end;
+  display: flex;
+  flex-direction: column;
+}
+
+.section {
+  width: 65%;
+  margin: auto;
+}
+
+.section.form {
+  height: calc(100% - 63px);
+}
+
+.departments {
+  margin: auto;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.departments span {
+  flex-grow: 1;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 10px;
+}
+
+.ed-tab, .biz-tab {
+  cursor: pointer;
+}
+
+.ed-text, .biz-text {
+  visibility: hidden;
+  margin: auto;
+  height: 0;
+  width: 90%;
+}
+
+.ed-text p, .biz-text p {
+  margin: 0;
+  height: 0;
+}
+
+.ed-text.active, .biz-text.active {
+  visibility: visible;
+  height: auto;
+}
+
+.ed-text.active p, .biz-text.active p {
+  margin: 1em 0;
+  height: auto;
+}
+
+.ed-tab.active {
+  font-weight: 700;
+  border-bottom: 3px solid #FFF;
+}
+
+.biz-tab.active {
+  font-weight: 700;
+  border-bottom: 3px solid #000;
+}
+
+.inline-link {
+  display: inline-block;
+  margin: 0;
+}
+
+@media screen and (max-width: 1200px) {
+  .departments {
+    flex-wrap: wrap;
+  }
+
+  .departments span {
+    width: 33%;
+  }
+
+  .ed-text, .biz-text {
+    width: 100%;
+  }
+}
+
+@media screen and (max-width: 800px) {
+  .section {
+    width: 100%;
+  }
+}
+
+@media screen and (max-width: 450px) {
+  h1 {
+    font-size: 2rem;
+  }
+
+  h2 {
+    font-size: 1.8rem;
+  }
+
+  h3 {
+    font-size: 1.2rem;
+  }
+
+  section {
+    min-height: calc(100vh - 40px);
+    height: auto;
+    width: calc(100vw - 30px);
+    padding: 20px 15px;
+  }
+
+  section.intro {
+    min-height: calc(100vh - 90px);
+  }
+
+  section.join {
+    height: calc(100vh - 40px);
+  }
+
+  .section.form {
+    height: calc(100% - 86px);
+  }
+}
+
+@media screen and (max-width: 425px) {
+  .biz-body .departments span {
+    width: 50%;
+  }
+
+  .join-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    bottom: 0px;
+    right: 0px;
+    left: 0px;
+    height: 40px;
+    padding: 10px;
+  }
+
+  .intro-text {
+    margin-bottom: 60px;
+  }
+}
```

Comment:
Please rework this into styled-components instead of writing the styles by class.